README.md

```diff
@@ -0,0 +1,113 @@
+| shadowflow.encryption.public-key                   | java.lang.String                                                                                                      | ""            | Base 64 encoded version of an X509 Public Key. Used in a Cipher with algorithm "RSA/ECB/OAEPWITHSHA-256ANDMGF1PADDING".                         |
+| shadowflow.encryption.noop                         | java.lang.Boolean                                                                                                     | false         | Disables encryption but encodes differences as Base64.                                                                                          |
+| shadowflow.flows[*].percentage                     | java.lang.Integer                                                                                                     | 0             | Percentage of how many calls should be compared in the shadow flow. Should be in the range of 0-100. Zero effectively disables the shadow flow. |
+
+
+## Configuring Shadow Flow Beans
+
+Shadow Flow beans can be configured in your Spring Boot application by defining properties in the `application.yml`
+file. Here are the steps to configure shadow flows:
+
+1. Define the shadow flows you want to use in your application under the `shadowflow.flows` property. 
+   In this example, `flow1` and `flow2` are defined as shadow flows (which you can rename to anything). 
+   And `percentage` represents a percentage number in the range of 0-100 of how many calls should be compared in the shadow flow. 
+   Zero effectively disables the shadow flow.
+
+```yaml
+shadowflow:
+  flows:
+    flow1:
+      percentage: 50
+    flow2:
+      percentage: 75
+```
+
+2. Configure the encryption options for the shadow flows under the `shadowflow.encryption property`. You can choose to use
+   a cipher or a public key for encryption.
+
+* To use a cipher, define the `shadowflow.encryption.cipher` property. The cipher should have a `secret` and an
+  `initialization-vector`. The secret should be a 16, 24, or 32-byte string and the `initialization-vector` should be a
+  12-byte string.
+* To use a public key, define the `shadowflow.encryption.public-key` property. The public key should be a Base 64 encoded
+  version of an X509 Public Key.
+
+Either set one of the following:
+
+Cipher example: exposes bean `defaultEncryptionService` of type `EncryptionService` 
+```yaml 
+shadowflow:
+    encryption:
+        cipher:
+            secret: "3d7e0c4f8fbbd8d8a79e76cabc8f4e24"
+            initialization-vector: "3d7e0c4f8fbb"
+```
+
+Public key example: exposes bean `publicKeyEncryptionService` of type `EncryptionService`
+```yaml 
+shadowflow:
+    encryption:
```

Comment:
I think YAML formatting is broken here, I know it's not important in README. But people can copy-paste it...

Reply:
What do you mean? What is broken?
https://github.com/rabobank/shadow-tool-spring-boot-starter/blob/feat/readme/README.md looks okay

Reply:
Well, `cipher` should be removed no I had another look.

Reply:
There's a bit of inconsistency in the number of space between the code blocks, but that doesn't make the yaml itself invalid indeed.

Reply:
I meant indents